File: submissions/sm_002_abdul/week_03/day_2/assignments/taxCalculator.js
var income=200000
if (income<250000){
    console.log('Income is below 250000,so no tax deducted')
}
else if ((income>250000)&&(income<500000)) {
    savings=50/100*income
    taxable_income=income-savings
    tax=10/100*taxable_income
    console.log("tax paid: ", tax)
}
else if ((income>500000)&&(income<1000000)) {
    savings=30/100*income
    taxable_income=income-savings
    tax=20/100*taxable_income
    console.log("tax paid: ",tax)
}
else {
    savings=10/100*income
    if (savings<50000){
        taxable_income=income-savings
        tax=30/100*taxable_income
        console.log("tax paid: ",tax)
    }
    else {
        savings=50000
        taxable_income=income-savings
        tax=30/100*taxable_income
        console.log("tax paid: ",tax)
    }
}

File: submissions/sm_112_soumik/week_13/day_5/tax_calculator.py
import math
amt = int(input("Enter your Income ! : "))
savings = int(input("Enter your savings ! :"))
rebate = 0
taxable_income = 0
intrest = 0


def tax_cal(amt, rebate, taxable_income, intrest, savings):
    if amt <= 500000:
        rebate = savings - savings * 0.5
    elif amt > 500000 and amt < 1000000:
        rebate = rebate = savings - savings * 0.3
    elif amt > 1000000 and amt < 10000000:
        rebate = rebate = savings - savings * 0.3
    else:
        rebate = rebate

    if rebate > 50000:
        rebate = 50000

    taxable_income = amt - rebate
    print(f"rebate is {math.floor(rebate)}")
    taxable_income = amt - rebate
    if taxable_income > 250000 and taxable_income < 500000:
        intrest = 250000 - taxable_income * 0.1
    elif taxable_income > 50000 and taxable_income < 1000000:
        intrest = (taxable_income - 500000) * 0.2 + 25000
    elif taxable_income > 1000000:
        intrest = (taxable_income - 1000000) * 0.3 + 25000 + 100000
    print(f'tax paid is {intrest} and rebate is {rebate}')


tax_cal(amt, rebate, taxable_income, intrest, savings)
